src/airport_gui.cpp

```diff
@@ -348,7 +348,7 @@ class BuildAirportWindow : public PickerWindowBase {
 				for (int i = 0; i < NUM_AIRPORTS; i++) {
 					const AirportSpec *as = AirportSpec::Get(i);
 					if (!as->enabled) continue;
-					for (uint8_t layout = 0; layout < as->num_table; layout++) {
+					for (uint8_t layout = 0; layout < static_cast<uint8_t>(as->layouts.size()); layout++) {
```

Comment:
would it be nicer to change `layout` to be a size_t, rather than have the cast? Might need some signature changes as well I suppose

Reply:
That does lead to a bit of a rabbit hole, when there's no need for for a size_t field further down.